### PR TITLE
Bump Django version to 1.11.20 to eliminate security risk

### PR DESCRIPTION
**Pull request by Ibracadabra05**
Django 1.11.x before 1.11.19, 2.0.x before 2.0.11, and 2.1.x before 2.1.6 allows Uncontrolled Memory Consumption via a malicious attacker-supplied value to the django.utils.numberformat.format() function.

For more context: https://nvd.nist.gov/vuln/detail/CVE-2019-6975

### CODE REVIEW

**Review by balanceiskey — APPROVED** [review]

**Comment by Ibracadabra05** [comment]
Alright, I think this is ready for review after validating my changes locally.

File: webapp/codesearch/settings.py [hunk 1]
```diff
@@ -93,7 +102,7 @@
 
 LOGGING = {
     'version': 1,
-    # 'disable_existing_loggers': False,
+    'disable_existing_loggers': False,
```
Comment (stantonk) [inline webapp/codesearch/settings.py:105]: ?

**Review by stantonk — APPROVED** [review]

**Comment by stantonk** [comment]
looks good, just had one question above out of curiosity

File: webapp/codesearch/settings.py (hunk 1 above)
Comment (Ibracadabra05) [reply webapp/codesearch/settings.py:105]: Some `INFO` level logs weren't showing after the bump. And that changed after this was uncommented. My suspicion is the default behavior for this attribute changed during one of the subsequent releases. @stantonk

Comment (stantonk) [reply webapp/codesearch/settings.py:105]: coolio

**Pull request merged**